backend/src/services/githubIntegrationService.ts

```diff
@@ -73,4 +79,105 @@ export default class GithubIntegrationService {
       url: repo.html_url,
     }))
   }
+
+  public static async findOrgDetail(org: string) {
+    const auth = await getGithubInstallationToken()
+    const { data } = await request('GET /orgs/{org}', {
+      org,
+      headers: {
+        authorization: `bearer ${auth}`,
+      },
+    })
+
+    return {
+      description: data.description || null,
+      github: data.html_url,
+      logoUrl: data.avatar_url,
+      name: data.login,
+      twitter: data.twitter_username || null,
+      website: data.blog || null,
+    }
+  }
+
+  public static async findOrgTopics(org: string, repos: { name: string }[]) {
+    const auth = await getGithubInstallationToken()
+
+    const topicSet = new Set<string>()
+
+    const topicPromises = repos.map(async (repo) => {
+      try {
+        const res = await request(`GET /repos/${org}/${repo.name}/topics`, {
+          headers: {
+            authorization: `bearer ${auth}`,
+          },
+        })
+
+        res.data.names.forEach((topic: string) => topicSet.add(topic))
+      } catch (err) {
+        console.error(`Failed to fetch topics for ${repo.name}:`, err.response?.data || err.message)
```

Comment:
NIT: It would be better to use the logger coming from LoggerBase here instead of the console